include/deal.II/base/time_stepping.templates.h

```diff
@@ -241,6 +241,232 @@ namespace TimeStepping
+          {
+            this->n_stages = 7;
+            this->b        = {{0.0941840925477795334,
+                        0.149683694803496998,
+                        0.285204742060440058,
+                        -0.122201846148053668,
+                        0.0605151571191401122,
+                        0.345986987898399296,
+                        0.186627171718797670}};
+            std::vector<double> ai;
+            ai = {{0.241566650129646868 + this->b[0],
+                   0.0423866513027719953 + this->b[1],
+                   0.215602732678803776 + this->b[2],
+                   0.232328007537583987 + this->b[3],
+                   0.256223412574146438 + this->b[4],
+                   0.0978694102142697230 + this->b[5]}};
+            this->a.push_back(ai);
+            break;
+          }
+        case (LOW_STORAGE_RK_STAGE9_ORDER5):
+          {
+            this->n_stages = 9;
+            this->b        = {{2274579626619. / 23610510767302.,
+                        693987741272. / 12394497460941.,
+                        -347131529483. / 15096185902911.,
+                        1144057200723. / 32081666971178.,
+                        1562491064753. / 11797114684756.,
+                        13113619727965. / 44346030145118.,
+                        393957816125. / 7825732611452.,
+                        720647959663. / 6565743875477.,
+                        3559252274877. / 14424734981077.}};
+            std::vector<double> ai;
+            ai = {{1107026461565. / 5417078080134.,
+                   38141181049399. / 41724347789894.,
+                   493273079041. / 11940823631197.,
+                   1851571280403. / 6147804934346.,
+                   11782306865191. / 62590030070788.,
+                   9452544825720. / 13648368537481.,
+                   4435885630781. / 26285702406235.,
+                   2357909744247. / 11371140753790.}};
+            this->a.push_back(ai);
+            break;
+          }
+        default:
+          {
+            AssertThrow(false,
+                        ExcMessage(
+                          "Unimplemented low-storage Runge-Kutta method."));
+          }
+      }
+    // compute ci
+    this->c.reserve(this->n_stages);
+    this->c.push_back(0.);
+    double sum_previous_bi = 0.;
+    for (unsigned int stage = 1; stage < this->n_stages; ++stage)
+      {
+        const double tmp = sum_previous_bi + this->a[0][stage - 1];
+        this->c.push_back(tmp);
+        sum_previous_bi += this->b[stage - 1];
+      }
+  }
+
+
+
+  template <typename VectorType>
+  double
+  LowStorageRungeKutta<VectorType>::evolve_one_time_step(
+    const std::function<VectorType(const double, const VectorType &)> &f,
+    const std::function<
+      VectorType(const double, const double, const VectorType &)>
+      & /*id_minus_tau_J_inverse*/,
+    double      t,
+    double      delta_t,
+    VectorType &y)
+  {
+    VectorType vec_ri;
+    VectorType vec_ki;
```

Comment:
what is the role of these vectors?

Reply:
Two auxiliary vectors that are required in each stage. 
The vector ki stores the evaluation of the differential operator, and ri is the RHS of the next stage.  
![image](https://user-images.githubusercontent.com/36655731/99881631-dc198880-2be8-11eb-82fc-5b272f5b4daf.png)

Reply:
okay, please document this.